ui/admin/app/components/form/managed-group/oidc/index.hbs

```diff
@@ -10,31 +10,29 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
-  <form.input
-    @value={{@model.type}}
-    @label={{t 'form.type.label'}}
-    readonly={{true}}
-    @disabled={{true}}
-  />
+  <InfoField @value={{@model.type}} disabled={{form.disabled}} as |F|>
```

Comment:
isn't `infofield` read-only? does this need a disabled attr?  we have a mix of these in the repo, can we make sure it is consistent and clean up if not needed?

Reply:
I understand it is not within the scope of this PR, a ticket is good with your findings

Reply:
@DhariniJeeva great point, I will put out a separate PR that addresses all of the other instances, but those in this PR are fixed in latest commit